src/NuProj.Targets/NuProj.targets

```diff
@@ -148,7 +150,7 @@
     <ProjectReference>
       <PackageDirectory>Lib</PackageDirectory>
       <TargetFramework></TargetFramework>
-      <Targets>$(ProjectOutputGroups)</Targets>
+      <ProjectOutputGroups>$(ProjectOutputGroups)</ProjectOutputGroups>
```

Comment:
In your commit, I see you're setting `Targets` to `$(ProjectOutputGroups)` which is defined entirely on its own rather than being based on the Common.targets' default list of targets to build. For this to work, we may need to add our own targets of interest to the default P2P targets list. That's probably important for more than just getting these failing tests to pass -- there could be other subtle behaviors in common.targets that rely on results from P2P targets that the add to the list.